logstash-core-event-java/build.gradle

```diff
@@ -92,11 +99,17 @@ idea {
 }
 
 dependencies {
-    compile 'com.fasterxml.jackson.core:jackson-core:2.7.1'
-    compile 'com.fasterxml.jackson.core:jackson-databind:2.7.1-1'
+    compile 'com.fasterxml.jackson.core:jackson-core:2.7.3'
+    compile 'com.fasterxml.jackson.core:jackson-databind:2.7.3'
+    compile 'com.esotericsoftware:kryo:3.0.3'
+
     provided 'org.jruby:jruby-core:1.7.22'
     testCompile 'junit:junit:4.12'
     testCompile 'net.javacrumbs.json-unit:json-unit:1.9.0'
+
+    runtime 'com.fasterxml.jackson.core:jackson-databind:2.7.4'
+    runtime 'org.jruby:jruby-complete:1.7.22'
```

Comment:
Sure. My gradle foo is weak. Hence the commenting out of the JRuby bits that you comments "Why?" on below.